pkg/config/config.go

```diff
@@ -145,6 +145,11 @@ func init() {
 	// Kube ApiServer
 	Datadog.SetDefault("kubernetes_kubeconfig_path", "")
 
+	// Datadog cluster agent
+	Datadog.SetDefault("cluster_agent_auth_token", "")
+	Datadog.SetDefault("cluster_agent_url", "")
+	Datadog.SetDefault("cluster_agent_kubernetes_service_name", "dca")
```

Comment:
also, didn't we want to bind `cluster_agent_auth_token` to the related env var?

Reply:
Yes, do we add the others ?

Reply:
Don't know the use cases of having the other ones as env vars, up to you :)

Reply:
We'll see when using it...